--- 35.Wisdom.orig	1994-07-13 19:53:02.000000000 +0200
+++ 35.Wisdom.mlxx	2023-01-18 09:52:34.321708600 +0100
@@ -5828,7 +5828,7 @@
 A(MARTA/NEI              V1  PAI3S  A(MARTA/NW
 U(/LHS                   N1  GSF    U(/LH
 GEW/DOUS                 A3H GSF    GEW/DHS
-EU)/QRAUSTA              A1B APN    EU)/QRASTOS
+EU)/QRAUSTA              A1B APN    EU)/QRAUSTOS
 SKEU/H                   N3E APN    SKEU=OS
 KAI\                     C          KAI/
 GLUPTA\                  A1  APN    GLUPTO/S
@@ -6341,7 +6341,7 @@
 OU)RANOU=                N2  GSM    OU)RANO/S
 PARE/SXES                VBI AAI2S  E)/XW            PARA
 AU)TOI=S                 RD  DPM    AU)TO/S
-A)KOPIA/TWS              D          A)KOPIA/STWS
+A)KOPIA/TWS              D          A)KOPIA/TWS
 PA=SAN                   A1S ASF    PA=S
 H(DONH\N                 N1  ASF    H(DONH/
 I)SXU/ONTA               V1  PAPASM I)SXU/W
